### PR TITLE
Fixed the ExpireDays error

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description
Fixed the ExpireDays error

### CODE REVIEW

File: cloud_governance/policy/common_policies/send_aggregated_alerts.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __get_es_data(self):
         :return:
         :rtype:
         """
-        current_date = (datetime.utcnow().date()).__str__()
+        current_date = (datetime.now(timezone.utc).date()).__str__()
```
Comment (ebattat) [inline cloud_governance/policy/common_policies/send_aggregated_alerts.py:33]: Can you search in the code if we have more places to fix it ?

Comment (athiruma) [reply cloud_governance/policy/common_policies/send_aggregated_alerts.py:33]: I'll open a separate PR on it. It showed me a warning sign on the page, so I changed it here.

Comment (ebattat) [reply cloud_governance/policy/common_policies/send_aggregated_alerts.py:33]: I think we should not wait, it should be part of upgrade to 4.12

Comment (ebattat) [reply cloud_governance/policy/common_policies/send_aggregated_alerts.py:33]: @athiruma , Pls open a PR for it ASAP !!
I will approve it this time.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**